_posts/0000-01-02-gmarcogen.md

```diff
@@ -0,0 +1 @@
+
```

Comment:
---
layout: slide
title: "Welcome to our second slide!"
---
I'm not sure if this worked
Use the left arrow to go back!